src/components/admin/ConfigDrop.tsx

```diff
@@ -0,0 +1,57 @@
+import { DropzoneArea } from 'material-ui-dropzone'
+import * as React from 'react';
+import {ClientConfig, ConvertToClientConfig} from '../../models/client_config';
+import '../../styles.css';
+
+class Props {
+  public updateClientConfig: (configObj: ClientConfig) => void;
+}
+
+interface State {
+  processing: boolean,
+  error: string
+}
+
+export class ConfigDrop extends React.Component<Props, State> {
```

Comment:
Could you add a couple tests for this? It would be neat to have an integration test where we simulate dropping a sample config (that we've committed to this repo) and parsing that config file to make sure everything is set correctly

Reply:
I added 3 new tests: a snapshot, simulating a drag event with an empty json file object, and simulating a drag event with a valid json file object.